Spec/Auth.swift

```diff
@@ -1903,6 +1903,32 @@ class Auth : QuickSpec {
                         done()
                     }
                 }
+                
+                tokenParams.clientId = "testId"
+                waitUntil(timeout: testTimeout) { done in
+                    rest.auth.authorise(tokenParams, options: authOptions) { tokenDetails, error in
+                        expect(error).to(beNil())
+                        guard let tokenDetails = tokenDetails else {
+                            XCTFail("TokenDetails is nil"); done(); return
+                        }
+
+                        expect(tokenDetails.clientId).to(equal("testId"))
+                        done()
+                    }
+                }
+                
+                tokenParams.clientId = nil
+                waitUntil(timeout: testTimeout) { done in
+                    rest.auth.authorise(tokenParams, options: authOptions) { tokenDetails, error in
+                        expect(error).to(beNil())
+                        guard let tokenDetails = tokenDetails else {
+                            XCTFail("TokenDetails is nil"); done(); return
+                        }
+                        
+                        expect(tokenDetails.clientId).to(beNil())
```

Comment:
> Also, method name is `authorize`

Ok, so I need to rename API from `authorise` to `authorize` (`z` letter is a diff) entirely in my current 0.8 scope, got it.